debezium-core/src/test/java/io/debezium/data/VerifyRecord.java

```diff
@@ -207,12 +207,12 @@ public static void hasValidKey(SourceRecord record, String pkField, int pk) {
     }
 
     /**
-     * Verify that the given {@link SourceRecord} is a {@link Operation#CREATE INSERT/CREATE} record without primary key.
```

Comment:
Hum, that's not what I head in mind here. This method should be used to ensure the record has _no_ PK. I.e. it's rarely used, only for those tests actually linked to handling of types without a PK. In all other cases, the overload expecting the PK field name and value should be used.

Reply:
Well, originally it asserted there *is* a PK and having a PK is definitely the more frequent case than the opposite.

Reply:
> having a PK is definitely the more frequent case than the opposite.

Yes, hence that method shouldn't be used most of the times.